test/test_rpc.py

```diff
@@ -3,19 +3,20 @@
 
 import sys
 import unittest
-from os import getenv
```

Comment:
Why does getenv need to move?

Reply:
It doesn't, but I just wanted to replicate how it looked before. We may want to talk about refactoring this to avoid in the future, which I've opened a discussion about in  #27129.